src/firmware_new/boards/arduino_uno/BUILD

```diff
@@ -0,0 +1,23 @@
+package(default_visibility = ["//visibility:public"])
+
+load(
+    "@platformio_rules//platformio:platformio.bzl",
+    "platformio_library",
+    "platformio_project",
+)
+
+platformio_library(
+    name = "echo_library",
```

Comment:
curious why it's suffixed by _library

Reply:
No real reason, but since its purpose is to be an example, i thought the name should have a clear indication of what it is an example of